hydra-filters/src/main/java/com/addthis/hydra/data/filter/value/ValueFilterHash.java

```diff
@@ -99,6 +101,10 @@ public ValueObject filterValue(ValueObject value) {
                 } catch (NoSuchAlgorithmException e) {
                     // ignore
                 }
+            case 5:
+                StringAndByteHashFunction mmh = new Murmur3HashFunction();
```

Comment:
It is wasteful to create a new instance of `Murmur3HashFunction` every time `filterValue` is called. A better approach is to use a singleton instance (e.g. it can be a static field of `Murmur3HashFunction` class). The singleton can also be used in `PluggableHashFunction`.